src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -69,6 +69,20 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.AgentUri"/>
         public const string AgentPort = "DD_TRACE_AGENT_PORT";
 
+        /// <summary>
+        /// Configuration key for windows named pipe where the Tracer can send traces.
+        /// Default value is null.
+        /// </summary>
+        /// <seealso cref="TracerSettings.ApmWindowsPipeName"/>
+        public const string ApmWindowsPipeName = "DD_APM_WINDOWS_PIPE_NAME";
+
+        /// <summary>
+        /// Configuration key for windows named pipe where the Tracer can send stats.
+        /// Default value is null.
+        /// </summary>
+        /// <seealso cref="TracerSettings.StatsWindowsPipeName"/>
+        public const string StatsWindowsPipeName = "DD_DOGSTATSD_WINDOWS_PIPE_NAME";
```

Comment:
Stats => Metrics
Windows Pipe => Named pipe
```suggestion
        /// Configuration key for the name of the named pipe where the Tracer can send metrics.
        /// Default value is <c>null</c>.
        /// </summary>
        /// <seealso cref="TracerSettings.MetricsNamedPipeName"/>
        public const string MetricsNamedPipeName = "DD_DOGSTATSD_WINDOWS_PIPE_NAME";
```